x/incentives/keeper/distribute_test.go

```diff
@@ -177,80 +185,107 @@ func (suite *KeeperTestSuite) TestSyntheticDistribute() {
 
 }
 
-// TestGetModuleToDistributeCoins tests the sum of coins yet to be distributed for all of the module is correct.
-func (suite *KeeperTestSuite) TestGetModuleToDistributeCoins() {
-	suite.SetupTest()
+// TestGetModuleToDistributeAndDistributedCoins tests the sum of coins yet to be distributed
+// and the distributed coins is correct.
+func (suite *KeeperTestSuite) TestGetModuleToDistributeAndDistributedCoins() {
+	lockOwner := sdk.AccAddress([]byte("addr1---------------"))
 
-	// check that the sum of coins yet to be distributed is nil
-	coins := suite.App.IncentivesKeeper.GetModuleToDistributeCoins(suite.Ctx)
-	suite.Require().Equal(coins, sdk.Coins(nil))
+	tests := []struct {
```

Comment:
nit: could you please add names to the tests? Just so it's clear to follow

Reply:
sure, I'll add some comment at each test cases, you think that's okay?